client/web/src/enterprise/batches/list/BatchChangeListPage.tsx

```diff
@@ -86,10 +86,11 @@ export const BatchChangeListPage: React.FunctionComponent<React.PropsWithChildre
     useEffect(() => telemetryService.logViewEvent('BatchChangesListPage'), [telemetryService])
 
     const isExecutionEnabled = isBatchChangesExecutionEnabled(settingsCascade)
+    const batchChangeLicensed = !!window.context.licenseInfo?.batchChanges
```

Comment:
Just to be consistent with how we name boolean variables, maybe:
```suggestion
    const isBatchChangesLicensed = !!window.context.licenseInfo?.batchChanges
```